gnovm/pkg/gnolang/op_expressions.go

```diff
@@ -78,8 +78,12 @@ func (m *Machine) doOpIndex2() {
 func (m *Machine) doOpSelector() {
 	sx := m.PopExpr().(*SelectorExpr)
 	xv := m.PeekValue(1)
-	res := xv.GetPointerTo(m.Alloc, m.Store, sx.Path)
-	*xv = res.Deref() // reuse as result
+	res := xv.GetPointerTo(m.Alloc, m.Store, sx.Path).Deref()
+	if debug {
```

Comment:
Not leftover, added after spending around half a day trying to figure out where this value was being changed in the stack, figuring out that the debug logs were not saying anything about it, and then realised that because this function does a "swap" in place of a pop+push, it didn't print anything in the logs :upside_down_face: 

I think this is useful to keep for any other adventurer who has to tread down these same treacherous roads I have, so I'd keep this.